### ISSUE TITLE
Present download size in humanized form

### ISSUE THREAD

**Issue by local-minimum** [JammerCore/JammerCore#2207]
format to one (or two) decimal closest unit (Gb, Mb, Kb, bytes) because it's quite hard to get a feel form e.g.

![bild](https://user-images.githubusercontent.com/8041100/194558797-20112ebe-5f64-4abe-8ef2-9309ee6dfde4.png)

**Comment by mikekasprzak** [comment]
Agreed. That entire page layout is disgusting to me, but I wanted to _at least_ make the information people needed was there.
